build.sh

```diff
@@ -23,6 +23,9 @@ datafiles="image-${arch}.tar.lzma sources.list"
 # Necessary development tool packages to be installed on build host
 devtools="debootstrap genisoimage p7zip-full squashfs-tools ubuntu-dev-tools"
 
+url_wallpaper="https://meilix-generator.herokuapp.com/uploads/wallpaper" # url heroku wallpaper
```

Comment:
Instead of downloading a wallpaper for the build, can't we have a default one in the repo itself